README.md

```diff
@@ -10,12 +10,13 @@ npm i fastify-jwt --save
 ```
 
 ## Usage
-Register it as plugin and then access it via `jwt`.  
-The api is the same of [jsonwebtoken](https://github.com/auth0/node-jsonwebtoken), refer to their documentation to find how use the utilities.
+Register as a plugin. This will decorate your `fastify` instance with the standard [jsonwebtoken](https://github.com/auth0/node-jsonwebtoken) methods `decode`, `sign`, and `verify`; refer to their documentation to find how to use the utilities. It will also register `request.jwtVerify` and `reply.jwtSign`. You must pass a `secret` when registering the plugin.
 
 ```js
 const fastify = require('fastify')
-fastify.register(require('fastify-jwt'), { secret: 'supersecret' }, err => {
+fastify.register(require('fastify-jwt'), { 
+  secret: 'supersecret' 
+  }, err => {
```

Comment:
Can you remove this callback? Is no longer supported :)

Reply:
oops missed this